src/pages/challenges.tsx

```diff
@@ -59,29 +67,40 @@ const Challenges: FC = () => {
                 Click here to Login
               </a>
             </div>
-          ) : <div><p>Something went wrong! Please contact admin</p></div>)
+          )
         }
+
         {
-        isLoading
-          ? (
-            <p>Loading...</p>
-          ) : (
-            <>
-              {
-                Object.keys(filteredChallenge).length > 0
-                  ? Object.keys(filteredChallenge).map((key) => (
-                    filteredChallenge[key].length > 0
-                    && (
-                    <Accordion open title={key} key={key}>
-                      {renderCardList(filteredChallenge[key], key, user.id)}
-                    </Accordion>
-                    )
+          !isLoading && error && 
```

Comment:
When does this condition occur?
Aren't we handling setting of isLoading to false in finally block?

Reply:
Made the change. Please re-check